### DISCUSSION TITLE
A double problem

### DISCUSSION THREAD

**Discussion by ghost** [geany/geany#3524] [Q&A]
Hello to all! 

I am a new user of Geany. The central idea is to use it for LaTeX and Scheme coding.

Problem #1 Upon start up I see the almost the half panel on the left (see attached picture). Although I change the margins, the next time is still there.
![pic1](https://github.com/geany/geany/assets/139242766/07b14f11-af67-4134-b0b5-573c6f25b750)

Problem #2 When I execute, e.g., Scheme code, the terminal is not cleared -at the beginning  (see attached picture)
![pic2](https://github.com/geany/geany/assets/139242766/b150ec42-416b-4dcd-8120-22e063d453cb)
In Edit > Preferences > Tools > Terminal , I have: /usr/bin/zellij -e "/bin/fish %c"
and
in Edit > Preferences > Terminal , I have: /usr/bin/zellij

My setting reads: 

terminal multiplexer: zellij
terminal emulator: alacritty
shell: fish 

Any ideas?

Thank you all in advance!

**Comment by elextr** [comment]
`#1` if you adjust the relationship between the sidebar pane and the edit pane it should be saved when Geany is quit.  If Geany is not quit (ie crash, or logout or shutdown with Geany open) it may not be saved.  This has changed to be more robust with the current git Geany and will be in the next version.

`#2` The terminal window is not cleared between executions.

**Reply by ghost** [reply]
Thank you for your kind reply.

Regarding #1, no it is not my case. The configuration is not saved -even though there is no crash, etc.

Regarding #2, my "problem" is that the terminal has many (2-3) empty line before and after the actual calculation/execution -as it is shown in the second picture. Should I change anything to my aforementioned configuration?

Thank you again.

**Comment by elextr** [comment]
`#1` saving windows size and position is required to be able to restore the pane positions, have you disabled them?

`#2` doesn't happen here with `/bin/bash` as shell, maybe its something "fishy" (tish boom :-)

**Reply by ghost** [reply]
How do I have them?
Thanks!

**Reply by ghost** [reply]
I do not think it is fish related...probably is zellij related.

**Comment by elextr** [comment]
`Edit->Preferences->General->Startup->Shutdown->` both `Save Window Size` and `Save Window Position` checked.

Its unlikely that any Geany contributors use zellij, or fish, I'm afraid you are likely to be on your own for how to configure them for use in the Geany virtual terminal.
